README.md

```diff
@@ -126,56 +142,87 @@ docker ps
+For a more detailed guide on local builds, please refer to the [tremor development docs](https://www.tremor.rs/community/development/quick-start).
 
-Tremor works by chaining operations that have inputs, outputs, and additional configuration. OnRamps - the operations that ingest data - take a unique role in this.
+### ARM/aarch64/NEON
 
-The documentation for different operations can found in the [docs](https://www.tremor.rs/docs/operations/cli). The `onramps` and `op` modules hold the relevant information.
+To run and compile with neon use:
 
-For each operation, the `Config` struct defines the parameters that can be passed to configure it, and the description holds additional details and examples.
+```bash
+RUSTCFLAGS="-C cpu-target=native" cargo +nightly build --features neon --all
+```
 
-### file sections
 
-#### `onramps`
+## Configuration
 
-A list that defines the OnRamps started in tremor. Each onramp lives in a separate thread. Along with its configuration, it has the key `pipeline` that defines the pipeline data from that onramp is sent to. At least one needs to be present.
+Tremor is configured using `.troy` files written in our own [Troy](https://tremor.rs/docs/0.12/language/troy/) language.
 
-```yaml
-onramps:
-  - onramp::file:
-      file: my-file.json
-      pipeline: main
-```
+Custom [Troy](https://tremor.rs/docs/0.12/language/troy/) modules can be loaded from any directory pointed to by the environment variable `TREMOR_PATH`.
+Directory entries need to be separated by a colon `:`.
+
+### Docker
+
+For use in docker [Troy](https://tremor.rs/docs/0.12/language/troy/) files should be mounted to `/etc/tremor/config`.
 
+Custom [Troy](https://tremor.rs/docs/0.12/language/troy/) modules and libraries should be mounted to `/usr/local/share/tremor`.
 ### Example
 
-Please look at the [demo](demo/configs/tremor) for a fully documented example
+This very simple example will consume lines from stdin and send them to stdout.
 
-### Configuration file usage in the docker container
+```troy
+define flow example
+flow
+  # import some common pre-defined pipeline and connector definitions
+  # to use here and save some typing
+  use troy::pipelines;
+  use troy::connectors;
 
-To use the configuration file as part of the Docker container mount the configuration files to `/etc/tremor/config`.
+  # create instances of the connectors and pipelines we need
+  create connector console from connectors::console;
+  create pipeline pass from pipelines::passthrough;
 
-## Local demo mode
+  # connect everything to form an event flow
+  connect /connector/console to /pipeline/pass;
+  connect /pipeline/pass to /connector/console;
+end;
+
+deploy flow example;
+```
+
+Run this example in file `example.troy` with docker:
+
+```console
+$ docker run -i -v"$PWD:/etc/tremor/config" tremorproject/tremor:0.12.0-rc.2
```

Comment:
should we encode a rc here? perhaps `:latest` would be better (that won't work right now but `-rc.2` won't be around for long either?